src/tipowof10.c

```diff
@@ -133,6 +160,25 @@ const vui128_t vtifrexpof10[] =
   (vui128_t) (__int128 ) 6103515625000000ll, /* 10**20 */
   (vui128_t) (__int128 ) 7629394531250000ll, /* 10**21 */
   (vui128_t) (__int128 ) 4768371582031250ll, /* 10**22 */
+
+#ifdef __clang__
```

Comment:
why? add a comment.

Reply:
// Clang does support constant folding for __int128
// Simply not posible to defind a const larger than __UINT64_MAX__
// For vector initialized can use two long long elements.